addon/modifiers/autoresize.js

```diff
@@ -42,21 +55,16 @@ export default class AutoresizeModifier extends Modifier {
     scheduleOnce('afterRender', this, 'resize');
   }
 
-  didInstall() {
-    // resize for initial value
-    this.scheduleResize();
+  modify(element, [value], named) {
+    this.el = element;
+    this.named = named;
+    this.value = value;
 
-    // resize on every input event
-    this.element.addEventListener('input', this.scheduleResize);
-  }
+    if (this.listener === null) {
+      this.listener = this.el.addEventListener('input', this.scheduleResize);
+    }
 
-  didUpdateArguments() {
-    // resize when arguments changes
     this.scheduleResize();
-  }
-
-  willRemove() {
-    // clean up
-    this.element.removeEventListener('input', this.scheduleResize);
+    registerDestructor(this, cleanup)
```

Comment:
Would be great if you could fix this linting issue. Would be interested to see if tests are passing.

```suggestion
    registerDestructor(this, cleanup);
```